packages/common/src/directives/ng_let.ts

```diff
@@ -6,56 +6,66 @@
  * found in the LICENSE file at https://angular.io/license
  */
 
-import {Directive, EmbeddedViewRef, Input, TemplateRef, ViewContainerRef} from '@angular/core';
+import {Directive, Input, TemplateRef, ViewContainerRef} from '@angular/core';
+
 
 /**
- * This is a directive to store data in a local variable
- * I developped it from angular's ngIf directive.
- * https://github.com/angular/angular/blob/6.1.1/packages/common/src/directives/ng_if.ts
- *
- * ## Storing result in a variable
- *
- * A common pattern is that we need to show a set of properties from the same object. If the
- * object is undefined, then we have to use the safe-traversal-operator `?.` to guard against
- * dereferencing a `null` value. This is especially the case when waiting on async data such as
- * when using the `async` pipe as shown in following example:
```

Comment:
I think, this part (21-30) still holds value. What do you think?

Reply:
Yeah, perhaps. It explains the issues around multiple async pipes. I was trying to get away from duplicating the content already available in `ngIf` 🤔

Maybe we can link to the original here? Otherwise, I'm not against putting it back.